apps/api/v2/src/lib/api-versions.ts

```diff
@@ -7,17 +7,24 @@ import {
   VERSION_2024_06_14,
   VERSION_2024_08_13,
   VERSION_2024_09_04,
+  VERSION_2025_07_11,
 } from "@calcom/platform-constants";
 
-export const API_VERSIONS_VALUES: VersionValue = API_VERSIONS as unknown as VersionValue;
+export const API_VERSIONS_VALUES = [...API_VERSIONS];
```

Comment:
Spread because API_VERSIONS is readonly and spread makes it mutable as required by VersionValue type

I removed _as unknown as VersionValue_ because I needed it to be an array,which it is.